amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/model/rma/type/RecordCategoryType.java

```diff
@@ -204,6 +194,30 @@ public Void doWork()
 
     }
 
+    /**
+     * Record Category move behaviour
+     *
+     * @see org.alfresco.repo.node.NodeServicePolicies.OnMoveNodePolicy#onMoveNode(org.alfresco.service.cmr.repository.ChildAssociationRef, org.alfresco.service.cmr.repository.ChildAssociationRef)
+     */
+    @Override
+    @Behaviour
+            (
+                    kind = BehaviourKind.CLASS,
+                    notificationFrequency = NotificationFrequency.FIRST_EVENT
+            )
+    public void onMoveNode(ChildAssociationRef oldChildAssocRef, ChildAssociationRef newChildAssocRef)
+    {
+        if (nodeService.getType(newChildAssocRef.getChildRef()).equals(TYPE_RECORD_CATEGORY) && dispositionService.getDispositionSchedule(oldChildAssocRef.getParentRef()) != null)
```

Comment:
In case of long conditions, it might be a good ideea to use multiple lines inside the `if` (eg. new line after `&&`).

Reply:
It's weird that the GS formatting template does not cover that. Thanks for noticing.

Reply:
not sure I understand this bit _dispositionService.getDispositionSchedule(oldChildAssocRef.getParentRef())_ , would the case when moving a sub-category that doesn't have retention applied to one that has be impacted by this?